### DISCUSSION TITLE
Is it possible to learn Fooocus new terms?

### DISCUSSION THREAD

**Discussion by Nightdragger2** [lllyasviel/Fooocus#1717] [Q&A] [answered]
Hello,

I know the question is general and not directly related to Fooocus, but this is the most appropriate place I could find to ask it.
Is there a way to learn Fooocus new terms?
There are terms where my Fooocus struggles to generate the correct image. For example, there are ships made from sci-fi books that should always look exactly the same.
Or terms that the AI doesn't actually do because they are usually not correct (e.g. a human with 4 arms and 8 fingers)

Can I somehow learn new terms from Fooocus? E.g. "Fooocus, this is a picture of the spaceship xy" or "Fooocus, this is a picture of a person with 4 arms and 8 fingers" and from then on Foocus knows what I want if I want a picture of a person with 4 arms.

**Comment by K3nt3L** [comment, marked as answer]
Foocus is just a software. Words (tokens in IA dialect) are learned during model training. The words are in the models, or LORAs.

Check this for better understanding :

https://stable-diffusion-art.com/how-stable-diffusion-work/
